widgets/common/src/BlueprintActions.ts

```diff
@@ -55,6 +56,15 @@ export default class BlueprintActions {
         return this.toolbox.getManager().doGet('/blueprints?_include=id', { params });
     }
 
+    doGetUploadedBlueprints(params?: Record<string, any>) {
+        return this.toolbox
+            .getManager()
+            .doGet('/blueprints?_include=id,state', { params })
+            .then(response =>
+                response.items.filter((blueprint: { state: string }) => BlueprintActions.isUploaded(blueprint))
+            );
```

Comment:
Wouldn't it be easier to do the filtering in a call?
```suggestion
            .doGet('/blueprints', { _include: 'id,state', state: BlueprintActions.CompletedBlueprintStates.Uploaded, ...params })
```